File: oauth2-sso-demo/authorization/src/main/kotlin/com/sleekydz86/oauth/global/application/user/UserSensitivePayload.kt
package com.sleekydz86.oauth.global.application.user

data class UserSensitivePayload(
    val displayName: String?,
    val role: String,
    val status: String,
    val createdAt: String,
    val passwordChangedAt: String,
    val passwordChangeFailCount: Int,
    val loginFailCount: Int,
    val lastLoginAt: String?,
    val passwordExpired: Boolean,
    val daysUntilPasswordChange: Long,
)

File: ko_chat/frontend/src/types/user/user-sensitive-payload.ts
export type UserSensitivePayload = {
  displayName: string | null
  role: string
  status: string
  createdAt: string
  passwordChangedAt: string
  passwordChangeFailCount: number
  loginFailCount: number
  lastLoginAt: string | null
  passwordExpired: boolean
  daysUntilPasswordChange: number
}
